ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/management-security.j2

```diff
@@ -110,4 +110,35 @@ management security
       crl {{ crl }}
 {%         endfor %}
 {%     endfor %}
+{%     if management_security.shared_secret_profiles is arista.avd.defined %}
```

Comment:
After removing the abstracted keys, please revisit the logic and build the tmp variable step by step instead of the many elif conditions. Also remember the other optimizations from other PRs like the automatic `arista.avd.defined` check inside `arista.avd.natural_sort`.

Reply:
The only command thing in all commands is
 "secret " ~ secret.name ~ " " ~ secret.secret_type | arista.avd.default("7") ~ " " ~ secret.secret | arista.avd.hide_passwords(hide_passwords).

Created temp variable for this part of command and moved other conditions into if statements.